src/sass/layout/_header.scss

```diff
@@ -4,6 +4,7 @@
   //   position: relative;
   display: flex;
   padding-top: 12px;
+  padding-bottom: 1px;
```

Comment:
why do you need this 1px?

Reply:
I also have `marging-top: -1` in hero because on some screen width it goes with 1px white line between these 2 blocks. Yaroslav did the same thing on the classes.